pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -283,20 +285,23 @@ func (b *awsElasticBlockStoreMounter) SetUpAt(dir string, fsGroup *int64) error
 			}
 		}
 		os.Remove(dir)
+		glog.Errorf("mount of disk %s failed: %v", dir, err)
 		return err
 	}
 
 	if !b.readOnly {
 		volume.SetVolumeOwnership(b, fsGroup)
 	}
 
+	glog.V(4).Infof("Successfully mounted %s", dir)
 	return nil
 }
 
 func makeGlobalPDPath(host volume.VolumeHost, volumeID string) string {
 	// Clean up the URI to be more fs-friendly
 	name := volumeID
 	name = strings.Replace(name, "://", "/", -1)
+	glog.V(4).Infof(path.Join(host.GetPluginDir(awsElasticBlockStorePluginName), "mounts", name))
```

Comment:
yeah, I think I was using that as debug and didn't remove it, don't think we need to log anything from here, I think I will remove